parts/language/lang_en.lua

```diff
@@ -858,19 +870,54 @@ return{
         {C.C,"Also try Orzmic!"},
         {C.C,"Also try osu!"},
         {C.C,"Also try Phigros!"},
+        {C.C,"Also try Puyo Puyo!"},
         {C.C,"Also try Rubik's cube!"},
         {C.C,"Also try Terraria!"},
+        {C.C,"Also try Touhou Project!"},
         {C.C,"Also try VVVVVV!"},
+        {C.G,"GREEN"},
```

Comment:
All color tips: I was against them because it adds no value to the player. It's not like the player is going to use any of the color codes, and if you want to showcase that your game can display that many colors, other colored tips do the job just as well.